src/main/java/BasicJavaHomeWork/Exercise_1.java

```diff
@@ -0,0 +1,17 @@
+package BasicJavaHomeWork;
+
+import java.util.Scanner;
+
+public class Exercise_1 {
+    public static void main(String[]args) {
+        System.out.print("Please type the seconds: ");
+        Scanner in = new Scanner(System.in);
+        int sec = in.nextInt();
+        int restSec = sec % 60;
+        int minutes = (sec / 60) % 60;
+        int hours = (sec /60) /60;
```

Comment:
you could change the hours calculation from (sec / 60) / 60 to sec / 3600

Reply:
Done